<launch>

  <arg name="raw_control_cmd" default="/control_cmd_raw"/>
  <arg name="filtered_control_cmd" default="/control_cmd_filtered"/>
  <arg name="control_filter_param"  default="$(find-pkg-share control_filter)/config/control_filter.param.yaml"/>
  <!-- control filter node -->
  <node pkg="control_filter" 
        exec="control_filter_node"
        name="control_filter_node"
        output="screen">

    <!-- パラメータ -->
    <param from="$(var control_filter_param)"/>
    <!-- トピックのremap -->
    <remap from="control_cmd_raw" to="$(var raw_control_cmd)"/>
    <remap from="control_cmd_filtered" to="$(var filtered_control_cmd)"/>

  </node>


</launch>